views/main/play.ejs

```diff
@@ -20,39 +20,39 @@
                 type="radio"
                 id="guess-a"
                 name="guess"
-                value="<%-gameName%>"
+                value="<%-gamesArray[0].gameName%>"
               />
-              <label for="guess-a"><%-gameName%></label>
+              <label for="guess-a"><%-gamesArray[0].gameName%></label>
```

Comment:
I think it's time to reduce the repetition here and render these elements dynamically by iterating over `gamesArray` and returning a `div` that has `input` and `label` elements in them with dynamic attributes.

Reply:
Was considering that but at the same time wasn't too sure if it was worth the time since its only 4 options. But i agree, inn the long run this would be a much better approach, will do it.